packages/lib/markdownIt.ts

```diff
@@ -0,0 +1,12 @@
+import MarkdownIt from "markdown-it";
+
+export const md = new MarkdownIt("default", { html: true, breaks: true, linkify: true });
+
+export function addListFormatting(html: string) {
+  return html
+    .replaceAll("<ul>", "<ul style='list-style-type: disc; list-style-position: inside; margin-left: 12px'>")
+    .replaceAll(
+      "<ol>",
+      "<ol style='list-style-type: decimal; list-style-position: inside; margin-left: 12px'>"
```

Comment:
why didn't you use tailwind here?

Reply:
Tailwind won't detect usage in this context as far as I know.

Reply:
isn't that path in the "content" config of tailwind?